sdk/core/azure-core/README.md

```diff
@@ -179,6 +179,21 @@ class MyCustomEnum(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     BAR = 'bar'
 ```
 
+#### Null Sentinel Value
```

Comment:
Just want to double check we expect this to be used by end user or developer or both?

Reply:
we expect this to be used by end user - if they want to pass an explicit null on the wire

Reply:
it's a rare edge case (which cloud events happen to allow), so decided to add a sentinel